frontend/src/components/layout/Layout.module.scss

```diff
@@ -7,17 +7,146 @@
+  --toastify-text-color-warning: #{$color-grey-50};
   --toastify-text-color-error: #{$color-white};
+
+  --toastify-toast-min-height: $layout-md;
+  --toastify-toast-max-height: $layout-md;
 }
 
+// Allows us to dynamically set the background color based on Toast container state
+$toastify-toast-close-button: var(--toast-close-button);
+
 .toast {
-  border-radius: $border-radius-md;
   text-align: center;
-  font-weight: 700;
+  font-weight: 400;
+  font-size: 13px;
+  line-height: 21px;
+  height: $layout-md;
+  border-radius: $border-radius-sm;
   font-family: $font-stack-base;
+  padding: 0;
+  max-width: $content-xl;
+  margin: 0 auto;
+
+  &:global {
+    &[class*="toast--error"],
+    &[class*="toast--info"] {
+      [class*="close-toast-button-icon"] {
+        color: #{$color-white};
+      }
+    }
+
+    &[class*="toast--success"],
+    &[class*="toast--warning"] {
+      [class*="close-toast-button-icon"] {
+        color: #{$color-grey-50};
+      }
+    }
+
+    &[class*="toast--error"] {
+      --toast-close-button: #{$color-red-60};
+
+      &:hover {
+        --toastify-color-error: #{$color-red-70};
+      }
+
+      &:active {
+        --toastify-color-error: #{$color-red-80};
+      }
+
+      &:focus {
+        --toast-close-button: #{$color-red-70};
+      }
+
+      &:disabled {
+        --toast-close-button: #{$color-red-80};
+      }
+    }
+
+    &[class*="toast--info"] {
+      --toast-close-button: #{$color-blue-50};
+
+      &:hover {
+        --toastify-color-info: #{$color-blue-60};
+      }
+
+      &:active {
+        --toastify-color-info: #{$color-blue-70};
+      }
+
+      &:focus {
+        --toast-close-button: #{$color-blue-60};
+      }
+
+      &:disabled {
+        --toast-close-button: #{$color-blue-70};
+      }
+    }
+
+    &[class*="toast--success"] {
+      --toast-close-button: #{$color-green-50};
+
+      &:hover {
+        --toastify-color-success: #{$color-green-60};
+      }
+
+      &:active {
+        --toastify-color-success: #{$color-green-70};
+      }
+
+      &:focus {
+        --toast-close-button: #{$color-green-60};
+      }
+
+      &:disabled {
+        --toast-close-button: #{$color-green-70};
+      }
+    }
+
+    &[class*="toast--warning"] {
+      --toast-close-button: #{$color-yellow-50};
+
+      &:hover {
+        --toastify-color-warning: #{$color-yellow-60};
+      }
+
+      &:active {
+        --toastify-color-warning: #{$color-yellow-70};
+      }
+
+      &:focus {
+        --toast-close-button: #{$color-yellow-60};
+      }
+
+      &:disabled {
+        --toast-close-button: #{$color-yellow-70};
+      }
+    }
+  }
+}
+
+.close-toast-button-container {
+  padding: $spacing-md;
+  position: relative;
+  border-radius: 0 $border-radius-sm $border-radius-sm 0;
+  background-color: $toastify-toast-close-button;
+  display: flex;
+  flex-direction: column-reverse;
+}
+
+.close-toast-button-icon {
+  width: 20px !important;
+  height: 20px !important;
```

Comment:
I set an ID for this custom close button to avoid fighting with Toastify styling and avoid using `!important` - hope that's an acceptable alternative. 

Also added the `declation-no-important` to stylelint.

Reply:
@Vinnl 100% agree about `!important` always needing an explanation. Thanks for adding, @lloan.